api/tools/src/main/java/org/pmiops/workbench/tools/institutions/Researcher.java

```diff
@@ -104,9 +115,12 @@ private InstitutionalRole parseInstitutionalRole(final String rawRole) {
       return InstitutionalRole.TRAINEE;
     }
 
-    // special case one particular user
-    // hopefully we will not need to add many such cases like this
-    if (rawRole.equals("Multispecialty Group Practice")) {
+    if (rawRole.startsWith("Data Analyst")) {
+      return InstitutionalRole.PROJECT_PERSONNEL;
```

Comment:
the description for this (see line 47, or original CAPS doc) includes Data Analyst